### PR TITLE
Add option to enable/disable bluetooth collection.

### PR DESCRIPTION
**Pull request by f2prateek**
Also disables bluetooth collection by default.

### CODE REVIEW

**Comment by f2prateek** [comment]
cc @tonyxiao

File: Analytics/Classes/Internal/SEGSegmentIntegration.m [hunk 1]
```diff
@@ -236,7 +238,7 @@ - (NSDictionary *)liveContext
     context[@"network"] = ({
         NSMutableDictionary *network = [[NSMutableDictionary alloc] init];
 
-        if (self.bluetooth.hasKnownState)
+        if (self.bluetooth && self.bluetooth.hasKnownState)
```
Comment (tonyxiao) [inline Analytics/Classes/Internal/SEGSegmentIntegration.m]: FWIW in Objc if `self.bluetooth` is `nil` then `self.bluetooth.hasKnownState` will evaluate to `NO` automatically. It's kind of an idiomatic objc thing, but of course both way works.

**Comment by tonyxiao** [comment]
lgtm

File: Analytics/Classes/Internal/SEGSegmentIntegration.m (hunk 1 above)
Comment (f2prateek) [inline Analytics/Classes/Internal/SEGSegmentIntegration.m]: Yeah I was considering not adding the `self.bluetooth` check - I don't have a strong preference either way so I ended up adding it.

**Pull request merged**